.dagger/bench.go

```diff
@@ -69,33 +73,38 @@ func (b *Bench) Specific(
 	// run benchmarks once with metrics tagged "prewarm" before running for real
 	// +optional
 	prewarm bool,
+	// notify this discord webhook on failure
+	// +optional
+	discordWebhook *dagger.Secret,
 ) error {
-	return b.bench(
+	return b.notifyOnFailure(ctx, b.bench(
 		ctx,
 		&benchOpts{
-			runTestRegex:  run,
-			skipTestRegex: skip,
-			pkg:           pkg,
-			failfast:      failfast,
-			timeout:       timeout,
-			race:          race,
-			count:         count,
-			testVerbose:   testVerbose,
-			prewarm:       prewarm,
+			runTestRegex:   run,
+			skipTestRegex:  skip,
+			pkg:            pkg,
+			failfast:       failfast,
+			timeout:        timeout,
+			race:           race,
+			count:          count,
+			testVerbose:    testVerbose,
+			prewarm:        prewarm,
+			discordWebhook: discordWebhook,
 		},
-	)
+	), discordWebhook, run)
 }
 
 type benchOpts struct {
-	runTestRegex  string
-	skipTestRegex string
-	pkg           string
-	failfast      bool
-	timeout       string
-	race          bool
-	count         int
-	testVerbose   bool
-	prewarm       bool
+	runTestRegex   string
+	skipTestRegex  string
+	pkg            string
+	failfast       bool
+	timeout        string
+	race           bool
+	count          int
+	testVerbose    bool
+	prewarm        bool
+	discordWebhook *dagger.Secret
```

Comment:
Don't think `discordWebhook` needs to be in here, it's accessed from an arg to `notifyOnFailure`?

Reply:
correct, missed this in a refactor